content/synthetics/_index.md

```diff
@@ -2,19 +2,67 @@
 title: Synthetics
 kind: documentation
 beta: true
-description: Synthetics
+description: documentation
 aliases:
   - /integrations/synthetics/
 further_reading:
 - link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
   tag: "Blog"
   text: "Introducing Datadog Synthetics"
+- link: "synthetics/browser_check"
+  tag: "Documentation"
+  text: "Configure a Browser Check"
+- link: "synthetics/uptime_check"
+  tag: "Documentation"
+  text: "Configure an Uptime Check"
 ---
 
 <div class="alert alert-warning">Synthetics is currently in beta. See the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics Request form</a>.</div>
 
-Datadog is pleased to announce the upcoming availability of Synthetics, a whole new layer of visibility on the Datadog platform. By monitoring your applications and API endpoints via simulated user requests, Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing as experienced by your users.
+## Overview
+
+Datadog Synthetics gives you a new layer of visibility on the Datadog platform. By monitoring your applications and API endpoints via simulated user requests and browser rendering, Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing as experienced by your users.
+
+{{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" responsive="true">}}
+
+## Search 
+
+Advanced search lets you query checks by any combination of checks attributes:
+
+* `title` and `message` - text search
+* `status` - Alert, No Data, Ok
+* `creator`
+* `region`
+* `muted`
+* `notification` 
+* `tags`
```

Comment:
I'm wondering if that bit could be a bit more detailed. For `region` for instance, it looks like we actually need to use `probe_dc` as a tag (tag:("probe_dc:aws:eu-central-1") basically). So this one could maybe be detailed a bit more like [some are for monitors](https://docs.datadoghq.com/monitors/manage_monitor/#find-the-monitors) but I guess I'm really being picky here... If they want to find out they could still use the facet and look at the search bar I guess.

Reply:
This tags should be removed soon as it's a "default" tag and is just confusing for the users. Indeed moving forward, using the facets and the search bar will be the way to go.